### PR TITLE
[12.0] [MIG] account_partner_reconcile

### PR DESCRIPTION
**Pull request by bjeficent**
Bump to version 12.0

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@bjeficent and @pedrobaeza This module belongs to https://github.com/OCA/account-payment
Should this PR move to the account-payment repo?

File: account_partner_reconcile/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# © 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (cubells) [inline account_partner_reconcile/__manifest__.py]: change `©` to `Copyright` please.

File: account_partner_reconcile/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#        (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Account Partner Reconcile",
+    'version': '12.0.1.0.0',
+    'category': 'Accounting',
+    'author': 'Eficent,'
+              'Odoo Community Association (OCA), ',
+    'website': 'https://github.com/OCA/account-payment',
```
Comment (cubells) [inline account_partner_reconcile/__manifest__.py]: I think this url must be `https://github.com/OCA/account-reconcile`

File: account_partner_reconcile/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# © 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (cubells) [inline account_partner_reconcile/models/res_partner.py]: change `©` to `Copyright` please.

**Review by cubells — CHANGES_REQUESTED** [review]
Minor changes

**Comment by bjeficent** [comment]
Comments attended, thanks for the review.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Review by pniederlag — APPROVED** [review]
button is there and working like intended. code looks good to me.

**Pull request merged**